app/controllers/admin/versions_controller.rb

```diff
@@ -1,17 +1,17 @@
 module Admin
   class VersionsController < Admin::BaseController
-    skip_authorization_check
+    load_resource :conference, find_by: :short_title
+    load_and_authorize_resource class: PaperTrail::Version
 
     def index
-      authorize! :index, PaperTrail::Version.new(item_type: 'User')
-      conf_ids_for_organizer = current_user.is_admin? ? Conference.pluck(:id) : Conference.with_role(:organizer, current_user).pluck(:id)
-      @versions = PaperTrail::Version.where(["conference_id IN (?) OR item_type = 'User'", conf_ids_for_organizer])
+      @conf_ids_with_role = current_user.is_admin? ? Conference.pluck(:short_title) : Conference.with_role([:organizer, :cfp, :info_desk], current_user).pluck(:short_title)
+
+      return if @conference.blank?
+      authorize! :index, PaperTrail::Version.new(conference_id: @conference.id)
```

Comment:
The abilities defined for `PaperTrail::Version` in [ability.rb](https://github.com/openSUSE/osem/pull/1530/files#diff-e9c04d21b8c4d777e4231c4c6ce5f8cbR233)  means that this will return false unless `item_type` is specified.
  
```ruby
cfp_role_ability.can? PaperTrail::Version.new(conference_id: @conference.id) #=> false
cfp_role_ability.can? PaperTrail::Version.new(conference_id: @conference.id, item_type: 'Event') #=> true
```
Therefore, only organizers will be able to access the `index` action for a specific conference.